### PR TITLE
Attributes

### PR DESCRIPTION
**Pull request by iocanel**

### CODE REVIEW

File: annotations/builder/src/main/resources/snippets/validate.txt [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  StringBuilder sb = new StringBuilder("Constraint Validations: ");
+  boolean first = true;
+  for (ConstraintViolation violation: violations) {
+     if (first) {
+        first = false;
+     } else {
+       sb.append(", ");
+     }
+     Object leafBean = violation.getLeafBean();
+     sb.append(violation.getPropertyPath() + " " + violation.getMessage() + " on bean: " + leafBean);
```
Comment (jimmidyson) [inline annotations/builder/src/main/resources/snippets/validate.txt]: Does it make sense to format with new lines?

Comment (iocanel) [inline annotations/builder/src/main/resources/snippets/validate.txt]: Its a matter of taste really!

I can see pros and cons either ways.

Comment (iocanel) [inline annotations/builder/src/main/resources/snippets/validate.txt]: @jstrachan: wdyt? I will release later tonight!

Comment (jstrachan) [inline annotations/builder/src/main/resources/snippets/validate.txt]: thanks - looks good. I figured the generation of the message might just be a helper function; to keep the code generation simpler (and result in less bytecode)? i.e. just have a static function which generates a canonical message given an `Iterable< ConstraintViolation>`

Comment (iocanel) [inline annotations/builder/src/main/resources/snippets/validate.txt]: Let me see if this can move inside BaseFluent as a static method.

**Pull request merged**